apis/v1alpha1/httproute_types.go

```diff
@@ -64,13 +64,16 @@ type HTTPRouteHost struct {
 	Rules []HTTPRouteRule `json:"rules"`
 
 	// ExtensionRef is an optional, implementation-specific extension to the
-	// "host" block.  The resource may be "configmaps" (omit or specify the
-	// empty string for the group) or an implementation-defined resource
+	// "host" block.  The resource may be "configmaps"  or an
+	// implementation-defined resource
 	// (for example, resource "myroutehosts" in group "networking.acme.io").
-	// Omitting or specifying the empty string for both the resource and
-	// group indicates that the resource is "configmaps".  If the referent
-	// cannot be found, the "InvalidRoutes" status condition on any Gateway
-	// that includes the HTTPRoute will be true.
+	//
+	// Valid resources for reference are up to the Controller. Examples
+	// include "configmaps"  or a custom resource (CRD).
```

Comment:
Nit: This seems to repeat the comment block above (`The resource may be "configmaps" or an implementation-defined resource (for example, resource "myroutehosts" in group "networking.acme.io"`). I think either are fine, we should just remove one.